relengapi/blueprints/clobberer/__init__.py

```diff
@@ -136,11 +136,31 @@ def branches():
     return [branch[0] for branch in branches]
 
 
+@bp.route('/lastclobber/all', methods=['GET'])
+@apimethod([rest.ClobberTime])
+def lastclobber_all():
+    "Return a sorted list of all clobbers"
+    session = g.db.session(DB_DECLARATIVE_BASE)
+    return session.query(ClobberTime).order_by(ClobberTime.lastclobber)
+
+
+@bp.route('/lastclobber/greater-than/<int:mintime>', methods=['GET'])
```

Comment:
This might be better as `/lastclobber/since/<int:mintime>` or even `/lastclobber/all?since=<int:mintime>`.